R/last_fit.R

```diff
@@ -65,42 +69,66 @@ last_fit <- function(object, ...) {
 last_fit.default <- function(object, ...) {
   empty_ellipses(...)
   msg <- paste0(
-    "The first argument to [last_fit()] should be either a ",
-    "formula, recipe, or workflow."
+    "The first argument to [last_fit()] should be either ",
+    "a model or workflow."
   )
   rlang::abort(msg)
 }
 
 #' @rdname last_fit
```

Comment:
For the soft deprecated ones, maybe remove them from the rd file of `last_fit()`? That way new users never know they existed in the first place, because the methods themselves won't show up. 

That might mean you don't have to document `@param model` too

Reply:
Addressed in 987683ecd785220ca3048106b658dcc198c58dae